v2g-liberty-cards/src/ping-card.ts

```diff
@@ -79,30 +93,42 @@ export class PingCard extends LitElement {
     return this._isResponding
       ? nothing
       : html`
-          <ha-alert alert-type="error">
-            <div class="error">${tp('error')}</div>
-          </ha-alert>
-          <p>
-            <ha-markdown breaks .content=${tp('error-subtext')}></ha-markdown>
-          </p>
-          <mwc-button @click=${this._restart}>${tp('restart')}</mwc-button>
+          <mwc-snackbar
+            ?open=${this._snackbarOpen}
+            labelText=${this._isRestarting ? tp('restarting') : tp('error')}
+            timeoutMs="-1"  <!-- Set timeout to -1 for persistent behavior -->
+            persistent  <!-- Ensure it remains visible until manually closed -->
+            @closed=${this._onSnackbarClosed}
+          >
+            <mwc-button slot="action" @click=${this._restart}>${tp('restart')}</mwc-button>
+          </mwc-snackbar>
         `;
   }
 
-  async _restart() {
+  private _onSnackbarClosed() {
+    this._snackbarOpen = false; // Snackbar closed event
+  }
+
+  async _restart(event: Event) {
+    event.stopPropagation(); // Prevent the click closing the snackbar
+    this._isRestarting = true; // Set flag to show restart message
+    this._snackbarOpen = true; // Ensure the snackbar stays open during restart
+
+    // Attempt the restart via the Home Assistant API
     await this.hass.callWS({
       type: 'supervisor/api',
       endpoint: `/addons/9a1c9f7e_v2g-liberty/restart`,
       method: 'post',
       timeout: null,
     });
-  }
 
-  static styles = css`
-    .error {
-      font-weight: bold;
-    }
-  `;
+    // After the restart:
+    // - Add a delay before the next ping to allow for the restart to take effect
+    // - After the delay start pinging as usual
```

Comment:
The `_ping` function already handles restarting the timer.
We do not want to start another sequence of timers, so I think we should remove this.

Reply:
Ohh, i thought this was only influencing the frequency of the current timer, not starting a new one. This should prevent the error to overwrite the "restarting..." message during restart.
Will research this.